frontend/src/index.js

```diff
@@ -3,13 +3,34 @@ import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import WebFont from 'webfontloader';
 import { addLocaleData } from 'react-intl';
-import de from 'react-intl/locale-data/de';
+
```

Comment:
Oh wow, we won't have all the languages at the beginning. Planned are Spanish, French, Turkish, Filipino, Indonesian, Swahili to start with.